### ISSUE TITLE
Ren Dystopia game freezes for importing dialogs

### ISSUE THREAD

**Issue by Choviics** [thesupersonic16/DALTools#34]
Hi, sorry to bother you again,

When you import dialogs into the script.pck file, sometimes the game freezes when trying to move from one scene to another. This problem seems to be random: sometimes the game transitions from one scene to the next smoothly, but other times it gets stuck in a scene and does not progress, even if you play again from the beginning. This glitch can occur several times and not always in the same parts of the game.

**Comment by thesupersonic16** [comment]
Please send me your Script.pck and which scene change you noticing the issue on.

**Comment by Choviics** [comment]
> Please send me your Script.pck and which scene change you noticing the issue on.

[Script.zip](https://github.com/user-attachments/files/16958262/Script.zip)
yes, here is my script, thanks for helping me

**Comment by thesupersonic16** [comment]
Where exactly does it freeze on when transitioning, does it freeze on a black screen or a loading screen? Also is this issue reproducable by constently replaying the same transition?

**Comment by Choviics** [comment]
> Where exactly does it freeze on when transitioning, does it freeze on a black screen or a loading screen? Also is this issue reproducable by constently replaying the same transition?

and the scene that freezes me is the following when natsumi does an action with the dishes and shido thinks something, if by any chance he continues he freezes when natsumi leaves and he should go to the scene where kotori looks for shido at the door, he freezes so that the images remain still and does not continue to the next, and apparently it is repeatable since it has been repeated many times in those scenes

**Comment by thesupersonic16** [comment]
What script file is this scene on? And how likely does the issue occor? Could you make a small video of the issue happening? Need to make sure I can reproduce it.

**Comment by thesupersonic16** [comment]
Ok I got the freeze, I had to progress a bit further.

**Comment by Choviics** [comment]
> Ok I got the freeze, I had to progress a bit further.

I just recorded the video XD, thank goodness it wasn't just me who had the problem, I thought I might have a problem with the font file but this already confirms that it is a problem with my script

**Comment by thesupersonic16** [comment]
The freeze is likely with BgmVolume missing its node. I have also found BgMove's definitions being incorrect. Pushed 0e6dcab24a581aab12c87c802bb5dbbeb5b907f9 which should fix the issues.

Since the instruction definitions were incorrect, you will need to reimport your translation in a clean Script.pck as your current pck is now corrupt.

Make sure to pull & build or grab the new build from GitHub Actions.

I have not checked if the new changes fixes your issue, but the fact there was a missing node is a likely cause.
